libraries/core_libs/storage/src/storage.cpp

```diff
@@ -423,6 +423,16 @@ std::deque<SortitionParamsChange> DbStorage::getLastSortitionParams(size_t count
   return changes;
 }
 
+SortitionParamsChange DbStorage::getParamsChangeForPeriod(uint64_t period) {
+  auto it =
```

Comment:
why not simply:

auto data = lookup(toSlice(period), Columns::sortition_params_change);
assert(!data.empty());

return SortitionParamsChange::from_rlp(dev::RLP(asBytes(data)));

Reply:
Will it return closest previous element? Because we have changes written only every interval blocks. For example we will have an record on key 50, but will search for 56. In that case `SeekForPrev` will definitely return what we need

Reply:
ah yeah you are right